scripts/whisper/README.md

```
@@ -9,5 +9,38 @@ Please see
+distil-whisper variants), you can discover alignment heads using:
+
+```bash
+python find_alignment_heads.py --model <model-name> --audio <test-audio.wav>
+```
+
+This script analyzes all attention heads and ranks them by:
+- **Monotonicity**: Whether attention peaks move forward as tokens are decoded
+- **Diagonal score**: Correlation with expected diagonal attention pattern
+
+Example output:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)